src/main/java/in/projecteka/consentmanager/user/UserRepository.java

```diff
@@ -35,7 +36,8 @@
                                 monoSink.error(new DbOperationError());
                                 return;
                             }
-                            var patientIterator = handler.result().iterator();
+                            RowSet<Row> result = handler.result();
+                            var patientIterator = result.iterator();
                             if (!patientIterator.hasNext()) {
```

Comment:
if (handler.result().iterator().hasNext()) {

Reply:
using the same thing at twice so, made as  var patientIterator = handler.result().iterator();